src/main/scala/scalatutorial/sections/ClassesVsCaseClasses.scala

```diff
@@ -6,170 +11,171 @@ import scalatutorial.aux.{BankAccount, Note}
 object ClassesVsCaseClasses extends ScalaTutorialSection {
 
   /**
-    * In the previous sections we have seen how case classes could be
-    * used to achieve information aggregation, and also how classes
-    * could be used to achieve data abstraction or to define stateful
-    * objects.
-    *
-    * What are the relationship between classes and case classes? How
-    * do they differ?
-    *
-    * = Creation and Manipulation =
-    *
-    * Remember the class definition of `BankAccount`:
-    *
-    * {{{
-    *   class BankAccount {
-    *
-    *     private var balance = 0
-    *
-    *     def deposit(amount: Int): Unit = {
-    *       if (amount > 0) balance = balance + amount
-    *     }
-    *
-    *     def withdraw(amount: Int): Int =
-    *       if (0 < amount && amount <= balance) {
-    *         balance = balance - amount
-    *         balance
-    *       } else throw new Error("insufficient funds")
-    *   }
-    * }}}
-    *
-    * And the case class definition of `Note`:
-    *
-    * {{{
-    *   case class Note(name: String, duration: String, octave: Int)
-    * }}}
-    *
-    * Let’s create some instances of `BankAccount` and `Note` and manipulate them:
-    */
+   * In the previous sections we have seen how case classes could be
+   * used to achieve information aggregation, and also how classes
+   * could be used to achieve data abstraction or to define stateful
+   * objects.
+   *
+   * What are the relationship between classes and case classes? How
+   * do they differ?
+   *
+   * = Creation and Manipulation =
+   *
+   * Remember the class definition of `BankAccount`:
+   *
+   * {{{
+   *   class BankAccount {
+   *
+   *     private var balance = 0
+   *
+   *     def deposit(amount: Int): Unit = {
+   *       if (amount > 0) balance = balance + amount
+   *     }
+   *
+   *     def withdraw(amount: Int): Int =
+   *       if (0 < amount && amount <= balance) {
+   *         balance = balance - amount
+   *         balance
+   *       } else throw new Error("insufficient funds")
+   *   }
+   * }}}
+   *
+   * And the case class definition of `Note`:
+   *
+   * {{{
+   *   case class Note(name: String, duration: String, octave: Int)
+   * }}}
+   *
+   * Let’s create some instances of `BankAccount` and `Note` and manipulate them:
+   */
   def creationAndManipulation(res0: String): Unit = {
     val aliceAccount = new BankAccount
-    val c3 = Note("C", "Quarter", 3)
+    val c3           = Note("C", "Quarter", 3)
```

Comment:
Agreed, it was just a proposal for code formatting. We just wanted to have a tool to automatize the code formatting.

Please, feel free to modify the code style here: https://github.com/scala-exercises/exercises-scalatutorial/blob/master/.scalafmt.conf

Thanks.